templates/EC2/jc-ec2-win.j2

```diff
@@ -217,34 +199,13 @@ Resources:
                   - ' -JcSystemsGroupId '
                   - !Ref JcSystemsGroupId
                   - ' -OwnerEmail '
-                  - !Ref OwnerEmail
+                  - !Ref JcUserId
                   - ' > C:\scripts\associate-jc-system.log'
-        tag_volume:
-          files:
-             "c:\\scripts\\tag-instance-volume.ps1":
-              source: "https://raw.githubusercontent.com/Sage-Bionetworks/infra-utils/v1.0.2/aws/tag-instance-volume.ps1"
-              mode: "0664"
-          commands:
-            01_tag_instance_volume:
-              command: !Join
-                - ''
-                - - 'Powershell.exe Set-ExecutionPolicy Bypass -Scope Process -Force;'
-                  - 'Powershell.exe C:\scripts\tag-instance-volume.ps1 '
-                  - ' -AwsRegion '
-                  - !Ref AWS::Region
-                  - ' -StackName '
-                  - !Ref AWS::StackName
-                  - ' -Department '
-                  - !Ref Department
-                  - ' -Project '
-                  - !Ref Project
-                  - ' -OwnerEmail '
-                  - !Ref OwnerEmail
-                  - ' > C:\scripts\tag-instance-volume.log'
     Properties:
       ImageId: !Ref AMIId
       InstanceType: !Ref InstanceType
       KeyName: !If [HasKeyName, !Ref KeyName, !Ref 'AWS::NoValue']
+      PropagateTagsToVolumeOnCreation: true
```

Comment:
When I follow the [link in the PR description](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-instance.html#cfn-ec2-instance-propagatetagstovolumeoncreation) I read, 

> PropagateTagsToVolumeOnCreation
Not currently supported by AWS CloudFormation.

Have you verified that this works?

Reply:
yes, i have verified that it works.  The doc is not up to date.